assets/components/src/web-preview/index.js

```diff
@@ -30,95 +30,152 @@ class WebPreview extends Component {
 	state = {
 		device: 'desktop',
 		loaded: false,
-		previewVisibility: false,
+		isPreviewVisible: false,
+		modalDOMElement: null,
 	};
 
+	/**
+	 * Create a div and append it to the body, so that the modal is on the top layer
+	 */
+	componentDidMount() {
+		const modalDOMElement = document.createElement( 'div' );
+		document.body.appendChild( modalDOMElement );
+		this.setState( { modalDOMElement } );
```

Comment:
Seems possibly weird to maintain the DOM element in state. Would this be opportunity for a React Ref? (It could be that Portal+Ref is not a good combination, just thinking aloud)

Reply:
That's true! I've changed it so the DOM element is a class field (as it is in the React docs)